scripts/common_startup.sh

```diff
@@ -128,7 +130,7 @@ if [ $SET_VENV -eq 1 ]; then
         echo "ERROR: A virtualenv cannot be found. Please create a virtualenv in $GALAXY_VIRTUAL_ENV, or activate one."
         exit 1
     fi
-elif [ $SET_VENV -eq -0 -a $CONDA_ALREADY_INSTALLED -eq 1 ]; then
+elif [ $SET_VENV -eq 0 -a $CONDA_ALREADY_INSTALLED -eq 1 ]; then
```

Comment:
@nsoranzo Yep, that would have avoided this problem too.  Not worried about it for this particular line now since we explicitly set it to 1 or 0 above, but yeah, +1 to overhauling it in another PR.